### ISSUE TITLE
Tibial tubercle apophysitis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11194]
**Preferred term label:**
Tibial tubercle apophysitis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lateral knee radiograph showing tibial tuberosity apophysis irregularity and fragmentation; tibial tubercle osteochondrosis; traction apophysitis of the tibial tubercle; Osgood-Schlatter's disease

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Osgood-Schlatter disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001345
- label: Tibial tubercle apophysitis
- parents: [Abnormal tibia morphology (HP:0002992)](https://hpo.jax.org/app/browse/term/HP:0002992)
- pmid: PMID:28723024
- def: Lateral knee radiograph showing tibial tuberosity apophysis irregularity and fragmentation; tibial tubercle osteochondrosis; traction apophysitis of the tibial tubercle.
- comment: The patellar tendon inserts at the tibial tubercle and consists of cartilaginous tissue. This is followed by ossification of the tibial tubercle at ages 10-12 in girls and at ages 12-14 in boys. It is during this stage of bone maturation when Osgood Schlatter disease develops. The prevailing theory is that there is repeated traction over the tubercle leading to microvascular tears, fractures, and inflammation; which then presents as swelling, pain, and tenderness.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11194
- synonyms: Osgood-Schlatter disease(NONE)
